stable/pipelines/templates/pipelines-hookhandler-statefulset.yaml

```diff
@@ -370,6 +373,63 @@ spec:
           {{- if .Values.pipelines.router.readinessProbe.enabled }}
           readinessProbe:
 {{ tpl .Values.pipelines.router.readinessProbe.config . | indent 12 }}
+        {{- end }}
+        - name: observability
+          image: {{ include "pipelines.getImageInfoByValue" (list . "pipelines" "observability" ) }}
+          imagePullPolicy: {{ .Values.pipelines.observability.image.pullPolicy }}
+          securityContext:
+            runAsNonRoot: false
+            allowPrivilegeEscalation: false
+            capabilities:
+              drop:
+                - NET_RAW
+          env:
+            - name: JF_OBSERVABILITY_PARENT_SERVICE
+              value: "hookhandler"
+            - name: JF_OBSERVABILITY_PARENT_PRODUCT
+              value: "pipelines"
+            - name: JF_SHARED_SECURITY_MASTERKEY
+              valueFrom:
+                secretKeyRef:
+                {{- if not .Values.pipelines.unifiedSecretInstallation }}
```

Comment:
Hi @imjasmeet, Please add support for external secret name

Reply:
https://jfrog-int.atlassian.net/browse/PIPE-14139 ticket raised as this is already released.